.github/workflows/dockerhub-latest-image.yml

```diff
@@ -46,4 +49,4 @@ jobs:
       - name: build and publish images
         env:
           VERSION: latest
-        run: make mp-image-${{ matrix.target }}
+        run: make mp-image-${{ matrix.target }} GOARCH=${{ matrix.arch }}
```

Comment:
Another option is putting the `GOARCH` to the `env`. Right?
Since we already use the `env`, I'd prefer that.

Not a big deal, how do you say?

Reply:
It maybe pretty when we add more args. OK, i will adjust it as you say.